packages/data-provider/test/utils/formatUtils.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+
+import { isArrayWithValues, isEmptyString, isValidIdNumber, uniqueCharacters } from '../../src/utils/formatUtils.js';
+
+describe('Format Utils tests', () => {
+	describe('Validate if input is a valid ID number', () => {
+		it('should return true if input is a valid number', () => {
+			const validNumber = 1;
+			const response = isValidIdNumber(validNumber);
+			expect(response).to.be.true;
+		});
+
+		it('should return false if input is a not a valid number', () => {
+			const invalidNumber = 'one';
+			const response = isValidIdNumber(invalidNumber);
+			expect(response).to.be.false;
+		});
+	});
+
+	describe('Extract unique characters on a String', () => {
+		it('should return unique characters on a string', () => {
+			const duplicateCharactersString = 'AAABBC';
+			const uniqueCharactersString = 'ABC';
+			const response = uniqueCharacters(duplicateCharactersString);
```

Comment:
What do we use this for? Seems cool...

Reply:
we use this function to generate system IDs using only unique characters when user sets a custom alphabet